### DISCUSSION TITLE
How to Rescan OCR on iPad

### DISCUSSION THREAD

**Discussion by raymond1138** [zsviczian/obsidian-excalidraw-plugin#2064] [Q&A] [answered]
I’m just getting started with Excalidraw and so far so good! My main use case has been using the OCR to convert my handwritten notes on iPad to text and it does a great job. In the instructional video, @zsviczian shows how to rescan a drawing with a modifier key on desktop. Is there a way to trigger the same functionality on iPad?

I know I can just switch over to desktop at some later point but it does take me out of my workflow and I’m hoping it’s something simple. I searched here, Reddit, the Obsidian Discord, and in the YouTube comments on the OCR instructional video and couldn’t find an answer.

**Comment by zsviczian** [comment, marked as answer]
The obsidian command palette is available also on tablets and phones. So the answer is yes. Look for the relevant OCR command in the command palette. You can access the command palette by pulling down from the top of the screen.

**Reply by raymond1138** [reply]
Perfect! I knew I was just missing something simple. Thanks for your hard work and the quick response!!
